lib/liveman_web/router.ex

```diff
@@ -2,28 +2,29 @@ defmodule LivemanWeb.Router do
   use LivemanWeb, :router
 
   pipeline :browser do
-    plug :accepts, ["html"]
-    plug :fetch_session
-    plug :fetch_flash
-    plug :protect_from_forgery
-    plug :put_secure_browser_headers
+    plug(:accepts, ["html"])
```

Comment:
Where are these formatting changes come from? 🤔 `mix.format` does not seem to require this so I am surprised.

Reply:
Reverted, I think my VS code `ElixirLS` plugin did it on save 😄